sdks/common/runner-api/src/main/proto/beam_runner_api.proto

```diff
@@ -755,9 +755,12 @@ message FunctionSpec {
   // passed as-is.
   string urn = 1;
 
+  // (Deprecated)
```

Comment:
There are tests that run expecting an any in this field. I'd like to do this with pairs of backwards-compatible changes. Specifically:

* Add and rename these fields. This permits the payload field to be populated everywhere.
* Remove existing usages of `any_param`, and use payload everywhere.
* Renumber `payload` to 2, and fill field 3 with a copy of the data (and deprecate)
* Import to dataflow and rebuild
* Remove the 3 field.